core/domain/skill_services.py

```diff
@@ -378,6 +378,44 @@ def get_skill_descriptions_by_ids(topic_id, skill_ids):
     return skill_id_to_description_dict
 
 
+def get_rubrics_by_skill_ids(topic_id, skill_ids):
+    """Returns a list of skill rubrics corresponding to given skill ids.
+
+    Args:
+        topic_id: str. The id of the topic that these skills are a part of.
```

Comment:
If the skills_ids are all skills belongs to a topic the should we just rename the method to `get_all_rubrics_belongs_to_topic(topic_id)` or something around that?

Reply:
Moved it to topic_fetchers